tests/test_intercept.js

```diff
@@ -276,9 +260,11 @@ test('encoding', async t => {
 })
 
 test('filter path with function', async t => {
+  // Interceptor.filteringPath simply proxies to Scope.filteringPath, this test covers the proxy,
+  // testing the logic of filteringPath itself is done in test_scope.js.
   const scope = nock('http://example.test')
-    .filteringPath(path => '/?a=2&b=1')
     .get('/?a=2&b=1')
+    .filteringPath(() => '/?a=2&b=1')
```

Comment:
Just to make sure I'm understanding: this test is of `.filteringPath()` called on an Interceptor. There's an identical "filter path with function" test of when `.filteringPath()` is called on a scope, that's being copied to `test_scope`. Is that right?

Reply:
Correct.